### ISSUE TITLE
Add feature flag for FxA Leanplum A/B test [bz1400871]

### ISSUE THREAD

**Issue by bug-mirror** [mozilla/fxa-bugzilla-mirror#374]
From https://bugzilla.mozilla.org/show_bug.cgi?id=1400871

To help support FxA push notification A/B testing we need to add a flag that does not automatically display native permissions dialog when a user signs in or up. Instead, it would rely on Leanplum framework to prime the user on why they should accept push notifications.

We have started a feature doc that has details on the different types of messaging that will be used, ref: https://docs.google.com/document/d/1W7W5RxbijWAKGingzwltbXMXWjbUlFGcrCWT1U8QArs/edit

To ensure everything is working as expected, this should be tested on non release users and be disabled when Leanplum is disabled.

**Comment by vbudhram** [comment]
Blocked on getting information for Leanplum SDK from their support.

**Comment by shane-tomlinson** [comment]
Support message received! @vbudhram to check out the info.

**Comment by vbudhram** [comment]
Code has been given greenlight to be merged! Still blocked on getting custom template created in Leanplum site.

**Comment by vbudhram** [comment]
Since original bugzilla bug has been closed, closing this in favor of https://github.com/mozilla/fxa-bugzilla-mirror/issues/407.
